src/etc/inc/filter.inc

```diff
@@ -3100,8 +3100,9 @@ function filter_generate_user_rule($rule) {
 }
 
 function filter_rules_generate() {
-	global $config, $g, $FilterIflist, $time_based_rules, $GatewaysList, $tracker;
+	global $config, $g, $FilterIflist, $time_based_rules, $GatewaysList, $tracker, $vlanprio_values;
 
+	$wancfg = $config['interfaces']['wan'];
```

Comment:
Why is it using `'wan'` interface hardcoded here?

Reply:
Don't know... copy paste finger trouble perhaps.

It's Removed.

Reply:
I had a muppet moment... try the update.